src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -13,7 +14,7 @@ import {imageConst} from '../../../../components/Image';
 
 const HUMAN_ID_URL = 'https://www.human-internet.org';
 
-const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, textSvg}) => {
+const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, text}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `textSvg` to `text` in the props list is a significant one. This changes the expected input from an SVG component to a string. This could potentially break any other components that are using `BottomOverlay` and passing an SVG to the `textSvg` prop. Make sure all instances of `BottomOverlay` have been updated to reflect this change.

```diff
- const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, textSvg}) => {
+ const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, text}) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->